src/main/java/org/apache/sling/cpconverter/maven/mojos/ConvertCPMojo.java

```diff
@@ -188,6 +189,9 @@
     @Component
     private RepositorySystem repoSystem;
 
+    @Parameter(defaultValue = "${project.remotePluginRepositories}", readonly = true)
```

Comment:
The artifacts are not plugins. Rather use project.remoteProjectRepositories (https://blog.sonatype.com/2011/01/how-to-use-aether-in-maven-plugins/)

Reply:
Thanks @kwin - I updated this.